src/components/Modal/BaseModal.js

```diff
@@ -15,11 +15,15 @@ const propTypes = {
 
     /** The ref to the modal container */
     forwardedRef: PropTypes.func,
+
+    /** Whether we should make status bar transclucent */
```

Comment:
@Tushu17 I like this description for the prop from [react-native docs](https://reactnative.dev/docs/modal#statusbartranslucent-android)

```suggestion
    /** Whether the modal should go under the system statusbar. */
```

Reply:
+1

Reply:
@Tushu17 i think you forgot to push this change

Reply:
@rushatgabhane oh, Ig I hadn't save that change.